discord/member.py

```diff
@@ -830,3 +830,20 @@ async def remove_roles(self, *roles: Snowflake, reason: Optional[str] = None, at
             user_id = self.id
             for role in roles:
                 await req(guild_id, user_id, role.id, reason=reason)
+
+    def get_role(self, role_id: int) -> Optional[discord.Role]:
+        """Returns a role with the given ID from roles which the member is in.
```

Comment:
```suggestion
        """Returns a role with the given ID from roles which the member has.
```
I think it's a bit more common to say that a member *has* certain roles, rather than saying that the member is *in* those roles.

Reply:
Ditto